server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FeatureFlags.kt

```diff
@@ -0,0 +1,74 @@
+package dev.slimevr.tracking.trackers.udp
+
+import java.nio.ByteBuffer
+
+/**
+ * Bit packed flags, enum values start with 0 and indicate which bit it is.
```

Comment:
Wait it's a bit set... Okay, I understand now. I'm not sure if it's better than being an int list or not...